workflow/executor/executor.go

```diff
@@ -887,7 +887,14 @@ func (we *WorkflowExecutor) Wait() error {
 	annotationUpdatesCh := we.monitorAnnotations(ctx)
 	go we.monitorDeadline(ctx, annotationUpdatesCh)
 
-	err = we.RuntimeExecutor.Wait(mainContainerID)
+	_ = wait.ExponentialBackoff(retry.DefaultRetry, func() (bool, error) {
```

Comment:
You are ignoring the error here. It should return here and handle in below.

Reply:
I have followed the same pattern as the other calls to wait.ExponentialBackoff in the same file. The error is still getting assigned to the err variable and is not being ignored.